src/host/ut_host/CommandLineTests.cpp

```diff
@@ -449,21 +449,21 @@ class CommandLineTests
 
         VERIFY_SUCCEEDED(m_pHistory->Add(L"I'm a little teapot", false));
         VERIFY_SUCCEEDED(m_pHistory->Add(L"short and stout", false));
-        VERIFY_SUCCEEDED(m_pHistory->Add(L"inflammable", false));
+        VERIFY_SUCCEEDED(m_pHistory->Add(L"Inflammable", false));

```

Comment:
ideally this test would be updated to verify that it does *not* match `i` with `Inflammable`.. or something? Right?

Are users going to be surprised by this change? When did we make it case-insensitive?